zap/src/main/java/org/zaproxy/zap/control/AddOnCollection.java

```diff
@@ -69,13 +69,10 @@ public AddOnCollection(
                 AddOn.AddOnRunRequirements requirements =
                         addOn.calculateInstallRequirements(addOns);
                 if (requirements.hasDependencyIssue()) {
-                    if (logger.isDebugEnabled()) {
```

Comment:
Better keep, the call to `getDependencyIssue` does a lot.

Reply:
Isn't that done behind the scenes now? I thought that was part of the benefit to the Log4j v2 upgrade

Reply:
As is now the referenced method will be called always without this check, whether debug is enabled or not. If we used a lambda it wouldn't, but not sure it allows to use one in that position.